projects/plugins/jetpack/class.jetpack-gutenberg.php

```diff
@@ -706,9 +709,10 @@ public static function enqueue_block_editor_assets() {
 		}
 
 		$initial_state = array(
-			'available_blocks' => self::get_availability(),
-			'modules'          => $modules,
-			'jetpack'          => array(
+			'available_blocks'        => self::get_availability(),
+			'available_media_sources' => array( 'jetpack_app_media' => self::blocks_variation() === 'experimental' ),
```

Comment:
True. This could work! Initially I though it would be good to be very specific as to who will see this but I think we can go ahead but I think limiting it to the block experiments should work just fine.